### PR TITLE
Toggle

### PR DESCRIPTION
**Pull request by AnnaRybkina**
- Three new pages added
- Migrated docs from https://react.fabric-ds.io/toggle and https://vue.fabric-ds.io/forms - toggle part, see screenshot 👇 
image

New components pages can be found 👇 
image

### CODE REVIEW

File: docs/components/select/vue.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ app.use(Forms)
 
 > Use in one component and special imports
 ```js
-import { fSelect } from '@warp-ds/vue'
+import { wSelect } from '@warp-ds/vue'
```
Comment (AnnaRybkina) [inline docs/components/select/vue.md:11]: Sneaky sneak

**Review by Skadefryd — APPROVED** [review]
Nice :)

File: package.json [hunk 2]
```diff
@@ -7,9 +7,9 @@
     "build": "vitepress build docs"
   },
   "devDependencies": {
-    "@warp-ds/component-classes": "1.0.0-alpha.59",
+    "@warp-ds/component-classes": "1.0.0-alpha.60",
```
Comment (BalbinaK) [inline package.json:10]: I saw the [last commit](https://github.com/warp-ds/component-classes/actions/runs/4934098671/jobs/8818734826) didn't get released but we'll get there sooner or later with the next commits :D

**Review by BalbinaK — APPROVED** [review]
Wow, so much docs 🤩

**Pull request merged**